netCDF4/_netCDF4.c

```diff
@@ -2054,7 +2054,7 @@ static const char __pyx_k_Class_providing_an_interface_to[] = "\nClass providing
 static const char __pyx_k_FillValue_attribute_must_be_set[] = "_FillValue attribute must be set when variable is ";
 static const char __pyx_k_MFDataset_set_auto_maskandscale[] = "MFDataset.set_auto_maskandscale";
 static const char __pyx_k_The_vltypes_dictionary_maps_the[] = "The `vltypes` dictionary maps the names of\n    variable-length types defined for the `netCDF4.Group` or `netCDF4.Dataset` to instances of the\n    `netCDF4.VLType` class.";
-static const char __pyx_k_Users_jswhit_python_netcdf4_pyt[] = "/Users/jswhit/python/netcdf4-python/netCDF4/_netCDF4.pyx";
```

Comment:
Just curious, what is all this stuff about @jjhelmus? Constants pointing to files on your machine... Why does Cython do that?